CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ option(AF_BUILD_FORGE
 option(AF_WITH_NONFREE  "Build ArrayFire nonfree algorithms"   OFF)
 option(AF_WITH_LOGGING  "Build ArrayFire with logging support" ON)
 option(AF_WITH_STACKTRACE  "Add stacktraces to the error messages." ON)
+option(AF_CACHE_KERNELS "Enable caching kernels on disk"       ON)
```

Comment:
This option will not define anything when building ArrayFire. You will need to add the definition here:

https://github.com/arrayfire/arrayfire/blob/master/CMakeLists.txt#L221

Perhaps name it CACHE_KERNELS_TO_DISK? because we cache kernels in memory but its not persistent across multiple processes.

Reply:
Done!